src/instructlab/data/generate.py

```diff
@@ -210,6 +211,27 @@ def generate(
         except Exception as exc:
             click.secho(f"Failed to start server: {exc}", fg="red")
             raise click.exceptions.Exit(1)
+
+    # figure out what model is running
+    try:
+        models = ilabclient.list_models(
+            api_base=api_base, http_client=http_client(ctx.params)
+        )
+
+        model = models.data[0].id if models is not None else None
```

Comment:
nit: This should probably gracefully handle a degenerate case where `data` is empty (no models are served).